fridgify/lib/utils/validator.dart

```diff
@@ -0,0 +1,86 @@
+class Validator {
```

Comment:
I found a few mistakes in the validation messages and i think we should have consistant wording and capitalization. However a few things are debatable. I point out grammar mistakes i catched and give a few suggestions on how i would write theses messages